crates/transaction-pool/src/validate/eth.rs

```diff
@@ -61,6 +61,62 @@ impl<Client, Tx> EthTransactionValidator<Client, Tx> {
     pub fn client(&self) -> &Client {
         &self.inner.client
     }
+
+    /// Returns the tracks activated forks relevant for transaction validation
+    pub fn fork_tracker(&self) -> &ForkTracker {
+        &self.inner.fork_tracker
+    }
+
+    /// Returns if there are EIP-2718 type transactions
+    pub fn eip2718(&self) -> bool {
+        self.inner.eip2718
+    }
+
+    /// Returns if there are EIP-1559 type transactions
+    pub fn eip1559(&self) -> bool {
+        self.inner.eip1559
+    }
+
+    /// Returns if there are EIP-4844 blob transactions
+    pub fn eip4844(&self) -> bool {
+        self.inner.eip4844
+    }
+
+    /// Returns if there are EIP-7702 type transactions
+    pub fn eip7702(&self) -> bool {
+        self.inner.eip7702
+    }
+
+    /// Returns the current max gas limit
+    pub fn block_gas_limit(&self) -> &AtomicU64 {
+        &self.inner.block_gas_limit
+    }
+
+    /// Returns the current tx fee cap limit in wei locally submitted into the pool
+    pub fn tx_fee_cap(&self) -> &Option<u128> {
+        &self.inner.tx_fee_cap
+    }
+
+    /// Returns the minimum priority fee to enforce for acceptance into the pool
+    pub fn minimum_priority_fee(&self) -> &Option<u128> {
+        &self.inner.minimum_priority_fee
+    }
+
+    /// Returns the setup and parameters needed for validating KZG proofs.
+    pub fn kzg_settings(&self) -> &EnvKzgSettings {
+        &self.inner.kzg_settings
+    }
+
+    /// Returns the config to handle [`TransactionOrigin::Local`](TransactionOrigin) transactions..
+    pub fn local_transactions_config(&self) -> &LocalTransactionConfig {
+        &self.inner.local_transactions_config
+    }
+
+    /// Returns the maximum size in bytes a single transaction can have in order to be accepted into
+    /// the pool.
+    pub fn max_tx_input_bytes(&self) -> &usize {
+        &self.inner.max_tx_input_bytes
+    }
```

Comment:
no need to return a ref here
```suggestion
    /// Returns the maximum size in bytes a single transaction can have in order to be accepted into
    /// the pool.
    pub fn max_tx_input_bytes(&self) -> usize {
        self.inner.max_tx_input_bytes
    }
```